_scripts/fetch-upstream-resources.sh

```diff
@@ -35,6 +35,7 @@ wget -O md_source/engine/api/v1.29/swagger.yaml https://raw.githubusercontent.co
 # Get a few one-off files that we use directly from upstream
 wget -O md_source/engine/reference/builder.md https://raw.githubusercontent.com/docker/docker/"$ENGINE_BRANCH"/docs/reference/builder.md || (echo "Failed engine/reference/builder.md download" && exit -1)
 wget -O md_source/engine/reference/run.md https://raw.githubusercontent.com/docker/docker/"$ENGINE_BRANCH"/docs/reference/run.md || (echo "Failed engine/reference/run.md download" && exit -1)
+wget -O md_source/edge/engine/reference/run.md https://raw.githubusercontent.com/docker/docker/v17.05.0-ce/docs/reference/run.md || (echo "Failed engine/reference/run.md download" && exit -1)
```

Comment:
This includes `v17.05.0-ce` which seems weird and likely to become wrong. Otoh, I have no idea what I'm talking about.